src/globus_sdk/_sphinxext.py

```diff
@@ -335,6 +335,36 @@ def gen_rst(self):
         yield ":ref:`how to make paginated calls <making_paginated_calls>`."
 
 
+class CopyParams(AddContentDirective):
+    has_content = True
+    required_arguments = 1
+    optional_arguments = 0
+
+    def gen_rst(self):
+        import globus_sdk
+
+        source_object_name: str = self.arguments[0]
+        path = source_object_name.split(".")
+        cur = globus_sdk
+        for element in path:
+            cur = getattr(cur, element)
+
+        if self.content:
+            content = iter(self.content)
+        else:
+            content = ()
+
+        for line in content:
+            if line.strip() == "<YIELD>":
+                break
+            yield line
+
```

Comment:
I don't see that this is getting used in this PR. Is it needed now or in the near-term?

Reply:
It is not being used right now. I could remove it but only if you feel strongly.

I wanted to make sure that it was possible to control where the static content is placed relative to the generated content. I've demonstrated that, so technically we could remove it with the knowledge that we can always reimplement something like this if it's needed.

As an alternative, we could define this bit of extension code to _only_ insert content where it finds a `<YIELD>` line (when there is content), so that we'd put `<YIELD>` into the usages with content and it would be used (trivially). That might be nice because it's more explicit about the ordering, rather than an "implicit append" rule.

Reply:
(Oh, and I'm not overly attached to `<YIELD>` as the syntax, if you have other ideas about that.)

Reply:
I'm not strictly tied to "using or losing" this code, but since it's not used it caught my eye.